src/Scripting/VM/VirtualMachine.cpp

```
@@ -0,0 +1,1281 @@
+VirtualMachine::VirtualMachine(
+	const Compiler::ConstantPool &constPool,
+	const std::reference_wrapper<Compiler::IGraphNodeRegistry> nodeRegistry,
+	AsTL::OpaqueExecCtx *execCtx,
+	const size_t vmStackSzKB,
+	const size_t callStackSzKB
+) :
+	m_allocVMStackSzKB(vmStackSzKB),
+	m_allocCallStackSzKB(callStackSzKB),
+	m_constPool(constPool),
+	m_nodeRegistry(nodeRegistry.get()),
+	m_execCtx(execCtx),
```

Cppcheck (2.21.0)

[style] The function 'Byteswap' is never used.

(unusedFunction)